src/Controller/UI/MultiSearchController.php

```diff
@@ -32,21 +32,27 @@ public function search(Request $request, MultiSearch $multiSearch, JsonSerialize
     {
         $queryString = $request->query->get('queryString');
         $page = $request->query->get('page');
+        $numberOfResultsPerPage = $request->query->get('perPage');
         $researchGroupFilter = $request->query->get('researchGroup');
+        $fundingOrganizationFilter = $request->query->get('fundingOrg');
+        $dataTypeFilter = $request->query->get('dataType');
 
         $searchOptions = new SearchOptions($queryString);
         $searchOptions->setCurrentPage($page);
         $searchOptions->setResearchGroupFilter($researchGroupFilter);
+        $searchOptions->setFundingOrgFilter($fundingOrganizationFilter);
+        $searchOptions->setDataType($dataTypeFilter);
+        $searchOptions->setMaxPerPage($numberOfResultsPerPage);
 
         $searchOptions->setFacets(array('researchGroup'));
 
         $searchResults = $multiSearch->search($searchOptions);
-
-        $groups = $groups = array(
+        $groups = array(
             'Default',
             'search',
             'result' => array(
                 'search',
+                'card',
```

Comment:
So, are we going to have only 'search'? or should we just leave it as is

Reply:
In this case we need search AND card.